modules/openxr/doc_classes/OpenXRAPIExtension.xml

```diff
@@ -75,6 +75,12 @@
 				Returns the id of the system, which is a [url=https://registry.khronos.org/OpenXR/specs/1.0/man/html/XrSystemId.html]XrSystemId[/url] cast to an integer.
 			</description>
 		</method>
+		<method name="is_environment_blend_mode_alpha_supported">
+			<return type="int" enum="OpenXRAPIExtension.OpenXRAlphaBlendModeSupport" />
+			<description>
+				Returns [enum OpenXRAPIExtension.OpenXRAlphaBlendModeSupport] denoting if [constant XRInterface.XR_ENV_BLEND_MODE_ALPHA_BLEND] is really support, emulated or not supported at all.
```

Comment:
nit: should be `is really supported`

Reply:
Ah, thanks! Fixed in my latest push

Reply:
Er, oops, this was already merged :-)

Reply:
Ok, here's a PR to fix the type-o: https://github.com/godotengine/godot/pull/88298